apps/api/src/app/notification-template/notification-template.controller.ts

```diff
@@ -174,7 +174,7 @@ export class NotificationTemplateController {
         steps: body.steps,
         notificationGroupId: body.notificationGroupId,
         active: body.active ?? false,
-        draft: body.draft ?? true,
+        draft: !body.active,
```

Comment:
always set `draft` as opposite of `value`. Ignore deprecated `body.draft`.

This is exactly what frontend does - but SDK users can set draft/active however they want what may lead them to bugs.